src/components/app/app.scss

```diff
@@ -332,3 +339,24 @@ ion-footer {
 [text-capitalize] {
   text-transform: capitalize;
 }
+
+// Float
+// --------------------------------------------------
+
+[pull-left],
```

Comment:
Could we do `pull-left`, `pull-right`, `pull-start`, and `pull-end` here and make them work like the `text-*` attributes do? Or is there a reason not to?